zuul/source/github.py

```diff
@@ -15,7 +15,7 @@
 import logging
 import time
 
-from zuul.model import PullRequest, Ref
+from zuul.model import NullChange, PullRequest, Ref
```

Comment:
The NullChange gets dropped in the return, so you could return anything here and it would basically get ignored.